licenses/ZEEBE-COPYING-HEADER.txt

```diff
@@ -0,0 +1,5 @@
+Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+one or more contributor license agreements. See the NOTICE file distributed
```

Comment:
:question: What is the NOTICE file? Is this generated by Maven and shipped in the JAR or should this live in the repo?

Reply:
Good question. It seems like something I'd have to add to the repo. I'm not quite sure what this is supposed to contain though. The one in Zeebe says:
```
Camunda
Copyright 2017-2019 Camunda Services GmbH
```
Which seems a bit outdated 😅